scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -969,6 +969,185 @@ cdef class TransformedDensityRejection(Method):
         return np.asarray(out).reshape(oshape)[()]
 
 
+cdef class SimpleRatioUniforms(Method):
+    r"""
+    SimpleRatioUniforms(dist, *, mode=None, pdf_area=1, domain=None, cdf_at_mode=None, random_state=None)
+
+    Simple Ratio-of-Uniforms (SROU) Method.
+
+    SROU is based on the ratio-of-uniforms method that uses universal inequalities for constructing
+    a (universal) bounding rectangle. It works for T-concave distributions with T(x) = -1/sqrt(x).
+
+    Parameters
+    ----------
+    dist : object
+        An instance of a class with ``pdf`` method.
+
+        * ``pdf``: PDF of the distribution. The signature of the PDF is
+          expected to be: ``def pdf(self, x: float) -> float``. i.e.
+          the PDF should accept a Python float and
+          return a Python float. It doesn't need to integrate to 1 i.e.
+          the PDF doesn't need to be normalized.
+
+    mode : float, optional
+        (Exact) Mode of the distribution. When the mode is ``None``, a slow
+        numerical routine is used to approximate it. Default is ``None``.
```

Comment:
When you say slow, could we make it fast by coding ourself something else in the future? The wording slow is a bit in contradiction with saying that the method is fast.

Reply:
> When you say slow, could we make it fast by coding ourself something else in the future?

Ideally, this method requires a user to provide the mode of the distribution. Although it can be approximated, it is not recommended to use this method when mode isn't available (unless it's easy to approximate the mode). That is why UNU.RAN throws a warning when mode is not set.

> The wording slow is a bit in contradiction with saying that the method is fast.

I also thought about this. We can make the parameter non-optional then. What do you think @tupui, @chrisb83?

Reply:
For me we just need to be clear about what slow and fast both means. Like if's slow, is it then slower than other methods. Adding some clarifications would be enough for me.